.azure-pipelines.yml

```diff
@@ -0,0 +1,248 @@
+
+    - bash: |
+        mkdir $BUILD_BINARIESDIRECTORY/bundle
+        cp target/release/{wasmtime,wasm2obj}.exe $BUILD_BINARIESDIRECTORY/bundle
+      displayName: Copy binaries (Win)
+      condition: and(eq( variables['Agent.OS'], 'Windows_NT' ), eq( variables['build_type'], 'release' ))
+    - bash: |
+        mkdir $BUILD_BINARIESDIRECTORY/bundle
+        cp target/release/{wasmtime,wasm2obj} $BUILD_BINARIESDIRECTORY/bundle
+      displayName: Copy binaries (*nix)
+      condition: and(ne( variables['Agent.OS'], 'Windows_NT' ), eq( variables['build_type'], 'release' ))
+    - task: ArchiveFiles@2
+      inputs:
+        rootFolderOrFile: $(Build.BinariesDirectory)/bundle
+        includeRootFolder: false
+        archiveType: 'zip'
+        archiveFile: '$(Build.ArtifactStagingDirectory)/wasmtime-windows-64bit-$(tagName).zip'
+      displayName: Archive files (Win)
+      condition: and(eq( variables['Agent.OS'], 'Windows_NT' ), eq( variables['build_type'], 'release' ))
+    - task: ArchiveFiles@2
+      inputs:
+        rootFolderOrFile: $(Build.BinariesDirectory)/bundle
+        includeRootFolder: false
+        archiveType: 'tar'
+        tarCompression: 'xz'
+        archiveFile: '$(Build.ArtifactStagingDirectory)/wasmtime-macos-64bit-$(tagName).tar.xz'
+      displayName: Archive files (macOS)
+      condition: and(eq( variables['Agent.OS'], 'Darwin' ), eq( variables['build_type'], 'release' ))
+    - task: ArchiveFiles@2
+      inputs:
+        rootFolderOrFile: $(Build.BinariesDirectory)/bundle
+        includeRootFolder: false
+        archiveType: 'tar'
+        tarCompression: 'xz'
+        archiveFile: '$(Build.ArtifactStagingDirectory)/wasmtime-linux-64bit-$(tagName).tar.xz'
+      displayName: Archive files (Linux)
+      condition: and(eq( variables['Agent.OS'], 'Linux' ), eq( variables['build_type'], 'release' ))
+    - task: PublishPipelineArtifact@1
+      inputs:
+        path: $(Build.ArtifactStagingDirectory)/
+        artifactName: 'bundle-$(Agent.OS)'
+      condition: eq( variables['build_type'], 'release' )
+
+- job: Publish
+  dependsOn: Build
+  condition: and(succeeded(), in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI'))
+  steps:
+  - checkout: none
+  - task: DownloadPipelineArtifact@1
+    inputs:
+      targetPath: $(Build.ArtifactStagingDirectory)
+  - script: |
+      echo "##vso[task.setvariable variable=tagName;]`echo $BUILD_SOURCEBRANCH | sed -e 's|refs/tags/||'`"
+    condition: startsWith(variables['Build.SourceBranch'], 'refs/tags/')
+  - task: GitHubRelease@0
+    inputs:
+      gitHubConnection: 'tschneidereit-releases'
+      repositoryName: 'tschneidereit/wasmtime'
+      action: 'edit'
+      target: '$(Build.SourceVersion)'
+      tagSource: 'manual'
+      tag: '$(tagName)'
+      title: 'Wasmtime $(tagName)'
+      assets: '$(Build.ArtifactStagingDirectory)/**'
+      isDraft: true
+      isPreRelease: true
```

Comment:
Should this and `isDraft` be `false` since this is an official release? Should the `isDraft` below for `master` be true?

Reply:
`isDraft` means that the release isn't publicly visible at all. `isPreRelease` is what indicates it's not an "official" release, I guess?

The idea was to have the `master` branch be a rolling release that happens immediately, whereas the tag releases would be drafts until a human pushes a button. However, I think for the time being we can just make them public immediately: it's better to have any releases than not to have them :)

For the time being these are all pre-releases, so I'll leave them marked as such.